<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirmwareUploaderWindow</class>
 <widget class="QDialog" name="FirmwareUploaderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Firmware Updater</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Loaded File:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblFilename">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLoadFile">
     <property name="text">
      <string>Load File</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Device Token:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtToken"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>CAN Bus:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spinBus"/>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Base Address:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="txtBaseAddr"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="lblProgress">
     <property name="text">
      <string>X of Y bytes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnStartStop">
     <property name="text">
      <string>Start Upload</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
